test/aida_web/controllers/session_controller_test.exs

```diff
@@ -282,16 +282,6 @@ defmodule AidaWeb.SessionControllerTest do
       assert DB.get_image(attachment_id).binary_type == "image/png"
       assert DB.get_image(attachment_id).source_url == nil
     end
-
-    test "does not send other file types to a session", %{conn: conn, bot: bot, session: session} do
```

Comment:
Fixed on cc0ae00. This test is working again because type validation was moved again to the method controller. But I wasn't able of making tests working for file size limitation.